packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -26,34 +26,6 @@ import { Presence } from '@microsoft/microsoft-graph-types-beta';
 
 export { PersonCardInteraction } from '../PersonCardInteraction';
 
-/**
- * Enumeration to define what parts of the person component render
- *
- * @export
- * @enum {number}
- */
-export enum PersonViewType {
-  /**
-   * Render only the avatar
-   */
-  avatar = 2,
-
-  /**
-   * Render the avatar and one line of text
-   */
-  oneline = 3,
-
-  /**
-   * Render the avatar and two lines of text
-   */
-  twolines = 4,
-
-  /**
-   * Render the avatar and three lines of text
-   */
-  threelines = 5
-}
-
```

Comment:
I agree with this change but it's unfortunately a breaking change. How about we deprecate this `PersonViewType` and update person to take in `PersonViewType | ViewType` so we avoid the breaking change?

Reply:
Makes sense. Question on this (I am updating the code), does this mean wherever we check this.view we would need to also check this.personView?

Reply:
And developer just can't use both at the same time...

Reply:
What if they do set both... which one takes priority 👯

Reply:
Anyways..... I took a little stab at it. Let me know if you like the implementation. Or else we can update it together on Friday.